generators/app/templates/boilerplate/eslintrc.js

```diff
@@ -38,26 +39,26 @@ module.exports = {
         'linebreak-style': [
             'off'
         ],
-        quotes: [
-            'error',
-            'single'
-        ],
+        // quotes: [
```

Comment:
This one resolves the single vs. double quote issue that was created by TS taking our single quotes as input, and emitting them as double quotes in some places.

The TSLint rule is already enforcing consistency, so no need to fight this one